test/e2e/virtual/workspaces/virtual_workspace_test.go

```diff
@@ -37,12 +40,15 @@ import (
 	kerrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	machineryutilnet "k8s.io/apimachinery/pkg/util/net"
+	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/wait"
 	kuser "k8s.io/apiserver/pkg/authentication/user"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
 
+	"github.com/kcp-dev/kcp/cmd/sharded-test-server/third_party/library-go/crypto"
 	virtualcommand "github.com/kcp-dev/kcp/cmd/virtual-workspaces/command"
 	virtualoptions "github.com/kcp-dev/kcp/cmd/virtual-workspaces/options"
 	tenancyv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/tenancy/v1alpha1"
```

Comment:
Is it now appropriate to remove the `t.Skip` on L170?

Reply:
Ack thanks, I just noticed that while testing locally - will remove, are any further changes needed to close out https://github.com/kcp-dev/kcp/issues/2056 ?

Reply:
Nothing further needed. Please mark this PR as also fixing 2056. Thanks

Reply:
Ok removed the skip but one of the test cases `a user sees his own workspaces at the root, but no other workspaces` is failing, looking into why that is